services/account_type/master_payer.yaml

```diff
@@ -274,58 +280,97 @@ Resources:
                   - 'arn:aws:s3:::${BucketName}-${Id}/*'
                   - { BucketName: !FindInMap [ Defaults, Cur, BucketName ],
                       Id: !Select [ 2, !Split [ '/', !Ref 'AWS::StackId' ] ] }
-          - Sid: CZCostMonitoring20191111
+          - Sid: CZProtect20190912
+            Effect: Deny
+            Action:
+              - aws-portal:*PaymentMethods
+              - aws-portal:*Account
+              - ec2:GetPasswordData
+              - ec2:GetConsoleOutput
+              - ec2:GetConsoleScreenshot
+              - ec2:GetLaunchTemplateData
+              - glue:GetConnection
```

Comment:
QUESTION: @silvexis, these are copied as is from the existing json policies. Do we need to update the Deny list too? Do we need to replace `aws-portal` with some new Deny Actions?

Reply:
ah, you know what, lets just delete the whole deny section from all our policies. They are unnecessary and just confuse people anyway...

Reply:
OK